utils/py/validate_kdf_responses.py

```diff
@@ -0,0 +1,467 @@
+            if responses_path.exists():
+                responses_by_version[version] = self._load_response_keys(responses_path)
+
+        # Compare and report mismatches
+        for version in ['legacy', 'v2']:
+            request_keys = requests_by_version.get(version, set())
+            response_keys = responses_by_version.get(version, set())
+            
+            # Requests without responses
+            missing_responses = request_keys - response_keys
+            for key in missing_responses:
+                self.warnings.append(ValidationError(
+                    f"src/data/kdf_responses/{version}/",
+                    "MISSING_RESPONSE",
+                    f"Request '{key}' exists but has no corresponding response data",
+                    f"{version}/{key}"
+                ))
+            
+            # Responses without requests
+            missing_requests = response_keys - request_keys
+            for key in missing_requests:
+                self.warnings.append(ValidationError(
+                    f"src/data/requests/{version}/",
+                    "MISSING_REQUEST", 
+                    f"Response '{key}' exists but has no corresponding request data",
+                    f"{version}/{key}"
+                ))
+
+    def _load_request_keys(self, requests_path: Path) -> set:
+        """Load all request keys from requests directory."""
+        request_keys = set()
+        try:
+            for json_file in requests_path.glob('*.json'):
+                with open(json_file, 'r', encoding='utf-8') as f:
+                    data = json.load(f)
+                    if isinstance(data, dict):
+                        request_keys.update(data.keys())
+        except Exception as e:
+            self.warnings.append(ValidationError(
+                str(requests_path),
+                "REQUEST_LOAD_ERROR",
+                f"Error loading request keys: {str(e)}"
+            ))
+        return request_keys
+
+    def _load_response_keys(self, responses_path: Path) -> set:
+        """Load all response keys from responses directory."""
+        response_keys = set()
+        try:
+            for json_file in responses_path.glob('*.json'):
+                with open(json_file, 'r', encoding='utf-8') as f:
+                    data = json.load(f)
+                    if isinstance(data, dict):
+                        response_keys.update(data.keys())
+        except Exception as e:
+            self.warnings.append(ValidationError(
+                str(responses_path),
+                "RESPONSE_LOAD_ERROR",
+                f"Error loading response keys: {str(e)}"
+            ))
+        return response_keys
+
+
+def main():
+    """Main execution function."""
+    parser = argparse.ArgumentParser(description='Validate KDF response JSON files')
+    parser.add_argument('--fix-minor-issues', action='store_true', 
+                       help='Automatically fix minor formatting issues')
```

Comment:
Is there any usage of `fix-minor-issues` arg / flag? I can't find any

Reply:
nope looks like a remnant. removed in https://github.com/KomodoPlatform/komodo-docs-mdx/pull/581/commits/e8f37596e1c653a08cbe55a5ee435610bf984ef1